src/main/java/seedu/address/logic/command/ClearCommand.java

```diff
@@ -19,8 +18,7 @@
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.setTravelPlan(new TravelPlan(new Name("Dummy"), new WanderlustDate("0"), new WanderlustDate("0")),
-                new TravelPlan(null, null, null));
+        model.setTravelPlanner(new TravelPlanner(null));
```

Comment:
Does this work? I think it cannot be null tho. "new TravelPlanner()" should work

Reply:
Hang on let me change it

Reply:
Ok changed